library/ssl_srv.c

```diff
@@ -3948,8 +3965,16 @@ static int ssl_write_new_session_ticket( mbedtls_ssl_context *ssl )
 
     MBEDTLS_SSL_DEBUG_MSG( 2, ( "=> write new session ticket" ) );
 
-    ssl->out_msgtype = MBEDTLS_SSL_MSG_HANDSHAKE;
-    ssl->out_msg[0]  = MBEDTLS_SSL_HS_NEW_SESSION_TICKET;
+    if( ssl && ssl->session_negotiate && ssl->session_negotiate->peer_cert ) {
+        ret = mbedtls_ssl_confirm_content_len( ssl, &ssl->out,
+            ssl->session_negotiate->peer_cert->raw.len + 4 + 12 + 2 + 16 + 10 +
```

Comment:
Please add comment explain what the values of 4,12,2,16,10 represent

Reply:
OK, done.